kie-platform-bom/pom.xml

```diff
@@ -641,6 +647,13 @@
       </dependency>
 
       <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>${version.org.springframework}</version>
+        <type>pom</type>
```

Comment:
We also need `<scope>import</scope>`. And I believe that the spring-bom needs to go _before_ the ip-bom, otherwise Maven will use the version from ip-bom (it simply takes the first place the version is, since both versions are provided via imported bom).